workflow/resources/datavzrd/go-enrichment-template.yaml

```
@@ -133,3 +133,12 @@ views:
               }  
             spec-path: ?input.vega_waterfall
             vega-controls: true
+  go_terms_scatter:
+    dataset: go_enrichment
+    desc: ?f"The y-axis represents the False Discovery Rate (FDR), while the x-axis shows the effect of the GO term for the {wildcards.model} model as determined by goatools.
+      The effect is determined as the average of all the absolute fold changes (all fold changes transformed to be >1) across all genes in a GO term (reported on a log2 scale).
+      Each point corresponds to a GO term.
+      Click on a GO term to highlight it in colour. 
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)